vng_api_common/utils.py

```diff
@@ -191,9 +191,14 @@ def request_object_attribute(
 ) -> str:
     client = get_client(url)
 
+    if not client:
+        return ""
```

Comment:
I haven't seen this function `request_object_attribute` before, it might be worth checking if this is actually used in any projects (or other libraries)